### PR TITLE
Addressing #127 - Reminder that data is not saved

### LINKED ISSUE #127: Reminder that data is not saved
**Is your feature request related to a problem? Please describe.**
Users have provided feedback stating that people are scared that data put in the form are being recorded and saved somewhere. This isn't the case and we note it on the first page but it may not be sufficient. 

**Describe the solution you'd like**
It would be useful to add reminders at the Results page and when closing the page (you will be losing all data).

### PR DESCRIPTION
**Pull request by ShadeWyrm**
Closes #127.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -6,6 +6,11 @@
         $t("linkProjectText")
       }}</a>
     </p>
+
+    <div class="alert alert-info">
```
Comment (nschonni) [inline src/views/Home.vue]: Is the alert dismissable? Not a big deal if it's not

Comment (ShadeWyrm) [reply src/views/Home.vue]: Not aware of any easy dismissable extension in WET.

Comment (nschonni) [reply src/views/Home.vue]: Only quick thing I found was https://wet-boew.github.io/wet-boew/demos/collapsible-alerts/collapsible-alerts-en.html

File: src/views/Home.vue [hunk 2]
```diff
@@ -8,7 +8,7 @@
     </p>
 
     <div class="alert alert-info">
-      <p>{{ $t("localSaveWarning") }}</p>
+      <p class="tinytext">{{ $t("localSaveWarning") }}</p>
```
Comment (nschonni) [inline src/views/Home.vue]: There are existing classes for this https://wet-boew.github.io/wet-boew-styleguide/design/sizing-en.html
Might also be an a11y issue if this is supposed to be important

**Pull request merged**